framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -828,6 +830,12 @@ public AnnotationMirror getAnnotationMirrorFromJavaExpressionString(
      */
     protected IdentityHashMap<Tree, Store> regularExitStores;
 
+    /**
+     * A mapping from methods (or other code blocks) to their exceptional exit store (used to check
```

Comment:
We don't check postconditions on exception exit. JML would have syntax for that, if I remember correctly. But here we only check postconditions on regular exits.